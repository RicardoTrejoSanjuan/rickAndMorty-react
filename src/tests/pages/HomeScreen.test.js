import React from 'react';
const { shallow } = require("enzyme");
const { default: HomeScreen } = require("../../pages/HomeScreen");

describe('Test en <HomeScreen />', () => {

    const title = 'The Rick and Morty API';
    const wrapper = shallow(<HomeScreen />);
    
    test('Dede mostrar el componente', () => {
        expect(wrapper).toMatchSnapshot();
    });

    test('Debe contener un <h1>', () => {
        const h3 = wrapper.find('h1');
        expect(h3.text()).toBe(title);
    });
});