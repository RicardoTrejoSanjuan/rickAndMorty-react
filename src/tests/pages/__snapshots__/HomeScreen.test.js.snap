// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test en <HomeScreen /> Dede mostrar el componente 1`] = `
<Fragment>
  <div
    className="container p-4"
  >
    <h1>
      The Rick and Morty API
    </h1>
    <SearchComponent
      setCharacter={[Function]}
    />
    <hr />
    <ol>
      <GridComponent
        character=""
        key=""
      />
    </ol>
  </div>
</Fragment>
`;
